community/kernel/src/main/java/org/neo4j/kernel/internal/StoreLocker.java

```diff
@@ -103,4 +107,9 @@ public void release() throws IOException
             storeLockFileChannel = null;
         }
     }
+
+    @Override
+    public void close() throws IOException {
+        release();
```

Comment:
What if instead of adding new `close` method we will just rename existing `release` to `close`?
Having two ways to do the same thing is always confusing and we do not have any interfaces or contracts to satisfy with current `release` methods